assets/src/blocks/Columns/ColumnsEditor.js

```diff
@@ -95,96 +95,96 @@ export const ColumnsEditor = ({ isSelected, attributes, setAttributes }) => {
   } = attributes;
 
   // Add className for existing blocks, based on columns_block_style attribute
-  useEffect(() => {
-    if (columns_block_style && !className) {
-      setAttributes({
-        className: `is-style-${columns_block_style}`
-      });
+  useEffect( () => {
+    if ( columns_block_style && ! className ) {
+      setAttributes( {
+        className: `is-style-${ columns_block_style }`,
+      } );
     }
-  }, []);
+  }, [] );
 
   // Update column block style based on className attribute
-  useEffect(() => {
-    const styleClass = getStyleFromClassName(className);
-    if (styleClass) {
-      setAttributes({
-        columns_block_style: styleClass
-      });
+  useEffect( () => {
+    const styleClass = getStyleFromClassName( className );
+    if ( styleClass ) {
+      setAttributes( {
+        columns_block_style: styleClass,
+      } );
     }
-  }, [className]);
+  }, [ className ] );
 
-  const { postType } = useSelect(select => ({
-    postType: select('core/editor').getCurrentPostType()
-  }), []);
+  const { postType } = useSelect( ( select ) => ( {
+    postType: select( 'core/editor' ).getCurrentPostType(),
+  } ), [] );
 
-  const { columnImages } = useSelect(select => {
-    let columnImages = [];
-    columns.forEach(column => {
-      if (column.attachment && column.attachment > 0) {
-        const media_details = select('core').getMedia(column.attachment);
-        if (media_details) {
-          columnImages[column.attachment] = select('core').getMedia(column.attachment).source_url;
+  const { columnImages } = useSelect( ( select ) => {
+    const columnImagesArray = [];
+    columns.forEach( ( column ) => {
+      if ( column.attachment && column.attachment > 0 ) {
+        const media_details = select( 'core' ).getMedia( column.attachment );
+        if ( media_details ) {
+          columnImagesArray[ column.attachment ] = select( 'core' ).getMedia( column.attachment ).source_url;
         }
       }
-    });
-    return { columnImages };
-  }, [columns]);
+    } );
+    return { columnImagesArray };
```

Comment:
Here it should be `return { columnImages: columnImagesArray };`

Reply:
Done